frame/staking/src/benchmarking.rs

```diff
@@ -113,8 +113,7 @@ pub fn create_validator_with_nominators<T: Config>(
 const USER_SEED: u32 = 999666;
 
 benchmarks! {
-	_{}
-
+	,
```

Comment:
Can this not be removed here and in the other files?

Reply:
Yes that is a very ugly hack to fix `local ambiguity: multiple parsing options: built-in NTs tt ('rest') or 1 other option` error thrown. Need to improve that :)

Reply:
The comma is a hack because he was running into some macro issues. He as asked me to help fix that, but maybe you can look and also make a quick suggestion?

The issue is related to parsing the `where` clause. I am thinking to just make a separate matching arm for this.

Reply:
I'm wondering why it works for the instance macro, but not for the normal one :thinking:

Reply:
it wasnt working :)